datavault-webapp/src/main/java/org/datavaultplatform/webapp/services/RestService.java

```diff
@@ -619,6 +618,10 @@ public RoleModel getIsAdmin() {
         return Arrays.asList(get(brokerURL + "/permissions/roleAssignments/user/" + userId, RoleAssignment[].class).getBody());
     }
 
+    public List<RoleAssignment> getRoleAssignmentsForRole(Long roleId) {
```

Comment:
Why `Long` vs a `long`?

Reply:
Kind of everywhere else it was a Long so I played along.

Reply:
But in this place indeed makes no sense